src/components/ConfirmationDialog.js

```diff
@@ -17,7 +17,7 @@ const ConfirmationDialog = ({
       sx={{
         display: 'flex',
         flexDirection: 'column',
-        height: '120px',
+        height: '140px',
```

Comment:
please do not chanhe the height directly, send a pop height, and by default its 120px